packages/config/config/devices/0x0346/motion_sensor_gen2.json

```
@@ -0,0 +1,195 @@
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Application Retries",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 5,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"3": {
+			"label": "Application Level Retries",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 1,
+			"maxValue": 96,
+			"defaultValue": 5,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"4": {
+			"label": "Led Indicator",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Don't show green",
+					"value": 0
+				},
+				{
+					"label": "Show green after Fault",
+					"value": 1
+				},
+				{
+					"label": "Show green after Intrusion and Intrusion clear",
+					"value": 2
+				}
+			]
+		},
+		"5": {
+			"label": "Occupancy Clear Delay",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"unsigned": true,
+			"allowManualEntry": true
+		},
+		"6": {
+			"label": "Instrusion Clear Delay",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"unsigned": true,
+			"allowManualEntry": true
+		},
+		"7": {
+			"label": "Standard Clear Delay",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"unsigned": true,
+			"allowManualEntry": true
+		},
+		"8": {
+			"label": "Motion Detection Mode",
+			"description": "Adjusts motion sensitivity",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 4,
+			"defaultValue": 3,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "0",
+					"value": 0
+				},
+				{
+					"label": "1",
+					"value": 1
+				},
+				{
+					"label": "2",
+					"value": 2
+				},
+				{
+					"label": "3",
+					"value": 3
+				},
+				{
+					"label": "4",
+					"value": 4
+				}
+			]
+		},
+		"9": {
+			"label": "Lighting Sensor",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Enabled",
+					"value": 0
+				},
+				{
+					"label": "Disabled",
+					"value": 1
+				}
+			]
+		},
+		"10": {
+			"label": "Lighting Delay",
+			"description": "Used to turn off lights when motion is no longer detected",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 60,
+			"defaultValue": 2,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"11": {
+			"label": "Supervisory Report Timeout",
```

```suggestion
			"label": "Supervision Report Timeout",
```
(that's a standardized CC name, Ring seems to like calling it "Supervisory" though)